### DISCUSSION TITLE
Weird persistent highlight after auto format

### DISCUSSION THREAD

**Discussion by mksonkar** [LazyVim/LazyVim#4712] [Q&A]
When I save the file, it auto formats the code into multiple lines and this weird highlight appears that doesn't go away with `:noh`. It only goes away after I close nvim and reopen it. (I'm using Prettier as formatter) 
![image](https://github.com/user-attachments/assets/68e577d9-510e-4c06-8be3-758f0c17fb02)
![2024-11-05-145955_1920x1080_scrot](https://github.com/user-attachments/assets/ec4e053a-f47b-4c7f-99b6-63c621551318)

**Comment by dpetka2001** [comment]
Did you complete this by accepting a snippet? If yes, this is a side-effect of Neovim's default `vim.snippet` engine and you should press `<Tab>` until you're out of the snippet for the highlight to disappear.

**Reply by mksonkar** [reply]
No I didn't accept any snippet. I typed the code manually in a single line. And when I pressed `<C-s>` to save it, it formatted the code in multiple lines, and the highlight appeared.

**Reply by dpetka2001** [reply]
Sorry, no idea then. Must be something with the formatter that you use would be my guess.
